<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="background">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>1</y>
      <width>971</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/pictures/resources/bg.png);</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>871</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgba(255, 255, 255, 0);
alternate-background-color: rgb(255, 255, 255);
gridline-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>License</string>
     </attribute>
     <widget class="QLabel" name="title_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>851</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 700 20pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>License</string>
      </property>
     </widget>
     <widget class="QLabel" name="instruction_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>861</width>
        <height>121</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>Once you clicked next you agree with this license. Please note that all the chracter and weapon pictures is not part of this project. They belongs to BWIKI of Genshin Impact, which is not allowed for commercial use. If you reject any part of the licence or word in this sentence, juet close the window.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextBrowser" name="license_browser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>861</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(33, 150, 243);</string>
      </property>
      <property name="source">
       <url>
        <string>qrc:/docs/resources/license</string>
       </url>
      </property>
     </widget>
     <widget class="QGraphicsView" name="next_from_page_one">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictures/resources/pushbutton_language.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_page_from_page_one_button">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;思源黑体&quot;;
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <zorder>title_1</zorder>
     <zorder>instruction_1</zorder>
     <zorder>next_from_page_one</zorder>
     <zorder>license_browser</zorder>
     <zorder>next_page_from_page_one_button</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Language Settings</string>
     </attribute>
     <widget class="QLabel" name="title_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>851</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 700 20pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>Language Settings</string>
      </property>
     </widget>
     <widget class="QLabel" name="instruction_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>861</width>
        <height>121</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>The language setting should be the same as your system. However, we only support English and Chinese. If your system preferred language is neither English nor Chinese, a fallback expected.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGraphicsView" name="next_from_page_two">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictures/resources/pushbutton_language.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_page_from_page_two_button">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;思源黑体&quot;;
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="back_to_page_one_bg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictures/resources/pushbutton_back.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previous_page_from_page_two_button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;思源黑体&quot;;
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <zorder>next_from_page_two</zorder>
     <zorder>title_2</zorder>
     <zorder>instruction_2</zorder>
     <zorder>next_page_from_page_two_button</zorder>
     <zorder>back_to_page_one_bg</zorder>
     <zorder>previous_page_from_page_two_button</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>User Profile</string>
     </attribute>
     <widget class="QLabel" name="title_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>851</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 700 20pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>User Profile</string>
      </property>
     </widget>
     <widget class="QLabel" name="instruction_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>861</width>
        <height>121</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>Please enter your basic user profile here. Please note that we do not use your information to get any data. The information you enter are only for displaying.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_username">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>601</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(149, 202, 243);
</string>
      </property>
     </widget>
     <widget class="QLabel" name="instruction_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="instruction_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>UID:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_uid">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>601</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(149, 202, 243);
</string>
      </property>
     </widget>
     <widget class="QLabel" name="instruction_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>Signature:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_signature">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>170</y>
        <width>601</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(149, 202, 243);
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previous_page_from_page_three_button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;思源黑体&quot;;
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="back_to_page_two_bg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictures/resources/pushbutton_back.png);</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="next_from_page_three">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictures/resources/pushbutton_language.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_page_from_page_three_button">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;思源黑体&quot;;
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <zorder>back_to_page_two_bg</zorder>
     <zorder>title_3</zorder>
     <zorder>instruction_3</zorder>
     <zorder>instruction_4</zorder>
     <zorder>instruction_5</zorder>
     <zorder>input_username</zorder>
     <zorder>input_uid</zorder>
     <zorder>instruction_6</zorder>
     <zorder>input_signature</zorder>
     <zorder>previous_page_from_page_three_button</zorder>
     <zorder>next_from_page_three</zorder>
     <zorder>next_page_from_page_three_button</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>User Avatar</string>
     </attribute>
     <widget class="QLabel" name="title_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>851</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 700 20pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>Avatar</string>
      </property>
     </widget>
     <widget class="QLabel" name="instruction_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>861</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>Please choose an avatar that you prefer. You can change it later any time.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="avatar_picker">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>851</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="back_to_page_three_bg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictures/resources/pushbutton_back.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="previous_page_from_page_four_button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;思源黑体&quot;;
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="next_from_page_four">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictures/resources/pushbutton_language.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_page_from_page_four_button">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;思源黑体&quot;;
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Record</string>
     </attribute>
     <widget class="QLabel" name="title_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>851</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 700 20pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
     </widget>
     <widget class="QLabel" name="instruction_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>861</width>
        <height>81</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>In this section you are going to enter your wish record. Please enter that in the new window. If there is no new window pop out, please find it in your task bar. If there is still nothing, simply open &quot;record-upd.exe&quot;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="previous_page_from_page_five_button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;思源黑体&quot;;
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="back_to_page_four_bg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictures/resources/pushbutton_back.png);</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="next_from_page_five">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictures/resources/pushbutton_language.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_page_from_page_five_button">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;思源黑体&quot;;
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <zorder>back_to_page_four_bg</zorder>
     <zorder>title_5</zorder>
     <zorder>instruction_8</zorder>
     <zorder>previous_page_from_page_five_button</zorder>
     <zorder>next_from_page_five</zorder>
     <zorder>next_page_from_page_five_button</zorder>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Finish</string>
     </attribute>
     <widget class="QLabel" name="title_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>851</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 700 20pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>Finish</string>
      </property>
     </widget>
     <widget class="QLabel" name="instruction_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>861</width>
        <height>81</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
font: 12pt &quot;思源黑体&quot;;</string>
      </property>
      <property name="text">
       <string>That is end of the setup. Please click &quot;next&quot; to start to use, if you wish to change your settings, please click &quot;back&quot;.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="previous_page_from_page_six_button">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;思源黑体&quot;;
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="back_to_page_five_bg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictures/resources/pushbutton_back.png);</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="next_from_page_six">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/pictures/resources/pushbutton_language.png);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_page_from_page_six_button">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>450</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;思源黑体&quot;;
background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <zorder>back_to_page_five_bg</zorder>
     <zorder>title_6</zorder>
     <zorder>instruction_9</zorder>
     <zorder>previous_page_from_page_six_button</zorder>
     <zorder>next_from_page_six</zorder>
     <zorder>next_page_from_page_six_button</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
